### DISCUSSION TITLE
Does the tool always select clips randomly? Can I have it edit chronologically?

### DISCUSSION THREAD

**Discussion by AKanoniOS** [scherroman/mugen#43] [Q&A]
I would like to make the tool create an edit, without reusing any clips, and all while using clips chronologically in the order they appear in the original file. Is the possible?

**Comment by scherroman** [comment]
Hey @AKanoniOS, chronological sampling isn't supported yet, though it has been [requested in the past (issue #10)](https://github.com/scherroman/mugen/issues/10). When thinking about this, I often wonder how the sampling should work given you'd still want the clips to be chosen randomly, just then sorted to be laid out chronological order. Though any given selected clip may not always pass the filters, which may then cause a cascading clash of not enough space between the previous and next clip to find a clip that passes without also having to change the neighboring clips, then their own neighboring clips, and so on. The filters complicate things, but it should still be doable, will keep this in mind when I continue work on the project. Pull requests are always welcome of course!
